communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "stimulusreflex.svg",
+      "title": "StimulusReflex",
+      "quote": "Discord has allowed us to gather over 1000 developers who discuss StimulusReflex and CableReady, as well as Rails, Stimulus and ViewComponent.",
+      "inviteCode": "XveN625",
```

Comment:
It looks like this invite code doesn't currently exist. I cannot identify your server without a valid invite code.

Reply:
I'm sorry to be a hassle.

https://discord.gg/XveN625

When I click the above with an Incognito window, it loads up correctly. We have people joining every day with this code.

What am I missing? 🤔

Reply:
Weird, maybe I did something wrong before, was reviewing 30+ PRs at once, its working now